app/interactions/assignment_interactions/update_complete.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# When an assignment is marked as complete (which only a Captain should be able
+# to do), we should apply the Assignment's rewards to the member.
+# We also need to check to see if the assignment was in a state where the reward
+# had already been applied. If so, something is horribly wrong and we want to
+# raise an error so we can track that down.
+# Rescue any errors, and always log what we did here.
+module AssignmentInteractions
+  # Class to handle logic after an Assignment has been updated to :complete
+  class UpdateComplete < BaseAssignmentInteraction
+    def initialize(options)
```

Comment:
I don't know that you need to define initialize here if all you're doing is calling super.